### DISCUSSION TITLE
Присвоение фильтрам пользователей

### DISCUSSION THREAD

**Discussion by savosik** [moonshine-software/moonshine#1146] [Ideas]
Интересно было бы чтобы админ мог в настройках предопределять фильтры, и назначать фильтры пользователям. Например нужно чтобы один пользователь видел только незавершённые заказы на завтра, второй завершенные вчера заказы, третий мог видеть только свои заказы, четвертый только чужие.  Количество возможных кейсов уходит за рамки метода query.  А так можно было бы всех пользователей развести по фильтрам, задав им определенные сферы видимости.

**Comment by lee-to** [comment]
Все это и так можно сделать, муншайн дает все возможности разработчику для этого
